tests/serverpod_test_server/lib/src/generated/protocol.dart

```diff
@@ -6752,11 +6752,6 @@ class Protocol extends _i1.SerializationManagerServer {
     if (t == List<int>) {
       return (data as List).map((e) => deserialize<int>(e)).toList() as T;
     }
-    if (t == _i1.getType<List<String>?>()) {
```

Comment:
FYI @SandPod The deduplication fix also found some other instance where `deserialize` code was generated for the same type multiple times (because return types and parameters where each iterated individually, without any deduplication).

Now we first fine all the types and then generated in a single loop.

I preserved the iteration order, so that now the diff in here is minimal.